api/dtos/authentication/TokenResponse.dto.ts

```diff
@@ -0,0 +1,3 @@
+export interface TokenResponseDto {
+  accessToken: string;
```

Comment:
![image](https://github.com/user-attachments/assets/2f53f20c-6c80-4db4-8269-d1023c8c0fa7)
tutaj mam tak, ale to zmienie zaraz bo powinien być camelCase

Reply:
kk